src/components/battleItemContainer.html

```diff
@@ -2,6 +2,17 @@
     <div class="card-header p-0" data-toggle="collapse" href="#battleItemContainerBody">
       <span>Battle Items</span>
     </div>
+    <button type="button" class="btn btn-sm btn-primary" 
+            style="position: absolute; right: 0px; top: 0px; width: auto; height: 41px;" 
+            data-bind="text: ItemHandler.multipliers[ItemHandler.multIndex()]"
+            onclick="ItemHandler.incrementMultiplier()"
+            oncontextmenu="ItemHandler.decrementMultiplier(); return false"
+            title="Controls"
+            data-toggle="popover"
```

Comment:
I think we should use a `tooltip` here to match the style of everything else.

Reply:
Ah, ok.

First, I thought that, but I found this popover and decided to try it.

Reply:
Actually, maybe we should leave it, and move the other things over to popovers instead.
It does look nicer IMO.

Reply:
maybe I just change the title to 'Use Multiplier' instead 'Controls', the problem is that this popover is relatively big than a tooltip